packages/web-app-files/src/index.js

```diff
@@ -138,5 +138,13 @@ export default {
       store.getters.configuration.server || window.location.origin,
       get(store, 'getters.capabilities.files.archivers', [])
     )
+    // FIXME: Remove mock data
+    thumbnailService.initialize(
+      get(store, 'getters.capabilities.files.thumbnail', {
```

Comment:
If guess we'll name it differently (like `preview` or something more generic), but that's not an issue for now. The way you implemented it allows a fix with minimal footprint ❤️

Reply:
Let's also check in the future if we can add supported mime types to the capabilities of oc10 as soon as the json schema for the capability is fixed.